.gitignore

```diff
@@ -4,7 +4,7 @@ db.sqlite3
 
 # Media ignores
 static
-static/media/
+media
```

Comment:
Surprised this was ever here, actually. Does the media file path in settings need to change, or do we keep that in local settings?  (I don't see it in the diff.)